context_test.go

```diff
@@ -998,7 +998,7 @@ func TestContextRenderIndentedJSON(t *testing.T) {
 	c.IndentedJSON(http.StatusCreated, H{"foo": "bar", "bar": "foo", "nested": H{"foo": "bar"}})
 
 	assert.Equal(t, http.StatusCreated, w.Code)
-	assert.Equal(t, "{\n    \"bar\": \"foo\",\n    \"foo\": \"bar\",\n    \"nested\": {\n        \"foo\": \"bar\"\n    }\n}", w.Body.String())
+	assert.JSONEq(t, "{\n    \"bar\": \"foo\",\n    \"foo\": \"bar\",\n    \"nested\": {\n        \"foo\": \"bar\"\n    }\n}", w.Body.String())
```

Comment:
This regresses the test. JSONEq will ignore the formatting but this test asserts that the body is indented JSON.

Reply:
Which testifylint rule was this? And with which version of testifylint?

I'd like to raise this an issue on the linter but I can't replicate the behaviour.

Reply:
> Which testifylint rule was this? And with which version of testifylint?

```
testifylint -V=full
/usr/local/bin/testifylint version devel comments-go-here buildID=b4feaeeee0471267f317aa4baaca3111987511befe6032bbd9b54f9f7a9f0923
```

Please note that the switch `-V` of testifylint is broken, there exists an [issue](https://github.com/Antonboom/testifylint/issues/175) for that already.

> I'd like to raise this an issue on the linter but I can't replicate the behaviour.

Please have a look at this [workflow run](https://github.com/deining/gin/actions/runs/12401028530/job/346194969789 which clearly shows the problem.